src/fundus/publishers/my/malay_mail.py

```diff
@@ -0,0 +1,35 @@
+import datetime
+from typing import List, Optional
+
+from lxml.cssselect import CSSSelector
+
+from fundus.parser import ArticleBody, BaseParser, ParserProxy, attribute
+from fundus.parser.utility import (
+    extract_article_body_with_selector,
+    generic_author_parsing,
+    generic_date_parsing,
+)
+
+
+class MalayMailParser(ParserProxy):
+    class V1(BaseParser):
```

Comment:
It would be great, if you could also add a function to extract keywords. This can be done very easily by using the keywords tag. MalayMail seems to support it within it's meta data and ld JSON